bluechi.spec.in

```diff
@@ -9,13 +9,15 @@
 %endif
 
 
-Name:		bluechi
-Version:	@VERSION@
-Release:	@RELEASE@%{?dist}
-Summary:	A systemd service controller for multi-nodes environments
-License:	LGPL-2.1-or-later AND CC0-1.0
-URL:		https://github.com/containers/bluechi
-Source0:	%{url}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+Name:		 bluechi
+Version:	 @VERSION@
+Release:	 @RELEASE@%{?dist}
+Summary:	 A systemd service controller for multi-nodes environments
+License:	 LGPL-2.1-or-later AND CC0-1.0
+URL:		 https://github.com/eclipse-bluechi/bluechi
+Vendor:   Eclipse BlueChi
```

Comment:
Please use TAB as a separator.

And wouldn't "Eclipse BlueChi project"  be better value?

Reply:
Done. 
I was using `CentOS` as an example - it doesn't have the project suffix.